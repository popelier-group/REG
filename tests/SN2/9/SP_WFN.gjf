# b3lyp/6-311++g(d,p) output=wfn

Single Point Energy

-1 1
 C                 -0.60253500   -0.00006100    0.00021300
 H                 -0.82525100   -0.89429000   -0.55693500
 H                 -0.82528000    0.92956800   -0.49559900
 H                 -0.82514100   -0.03547900    1.05324100
 Br                 1.59390200    0.00000300   -0.00001100
 Cl                -3.31902900    0.00002100   -0.00007000

frame.wfn
