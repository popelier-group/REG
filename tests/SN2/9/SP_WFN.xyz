
6

C       -0.481851    0.019927    0.098619
H       -0.543757   -0.535151    1.019335
H       -0.906161   -0.458937   -0.767582
H       -0.649620    1.080703    0.177889
Br        1.668284   -0.068296   -0.341233
Cl       -3.141136    0.128482    0.642459
