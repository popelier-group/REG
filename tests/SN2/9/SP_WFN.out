 Entering Gaussian System, Link 0=/opt/apps/apps/binapps/gaussian/g16c01_em64t_sandybridge/g16/g16
 Initial command:
 /opt/apps/apps/binapps/gaussian/g16c01_em64t_sandybridge/g16/l1.exe "/scratch/v69787ff/gau_temp_1513317.9/Gau-18027.inp" -scrdir="/scratch/v69787ff/gau_temp_1513317.9/"
 Default is to use a total of   2 processors:
                                2 via shared-memory
                                1 via Linda
 Entering Link 1 = /opt/apps/apps/binapps/gaussian/g16c01_em64t_sandybridge/g16/l1.exe PID=     18030.
  
 Copyright (c) 1988-2019, Gaussian, Inc.  All Rights Reserved.
  
 This is part of the Gaussian(R) 16 program.  It is based on
 the Gaussian(R) 09 system (copyright 2009, Gaussian, Inc.),
 the Gaussian(R) 03 system (copyright 2003, Gaussian, Inc.),
 the Gaussian(R) 98 system (copyright 1998, Gaussian, Inc.),
 the Gaussian(R) 94 system (copyright 1995, Gaussian, Inc.),
 the Gaussian 92(TM) system (copyright 1992, Gaussian, Inc.),
 the Gaussian 90(TM) system (copyright 1990, Gaussian, Inc.),
 the Gaussian 88(TM) system (copyright 1988, Gaussian, Inc.),
 the Gaussian 86(TM) system (copyright 1986, Carnegie Mellon
 University), and the Gaussian 82(TM) system (copyright 1983,
 Carnegie Mellon University). Gaussian is a federally registered
 trademark of Gaussian, Inc.
  
 This software contains proprietary and confidential information,
 including trade secrets, belonging to Gaussian, Inc.
  
 This software is provided under written license and may be
 used, copied, transmitted, or stored only in accord with that
 written license.
  
 The following legend is applicable only to US Government
 contracts under FAR:
  
                    RESTRICTED RIGHTS LEGEND
  
 Use, reproduction and disclosure by the US Government is
 subject to restrictions as set forth in subparagraphs (a)
 and (c) of the Commercial Computer Software - Restricted
 Rights clause in FAR 52.227-19.
  
 Gaussian, Inc.
 340 Quinnipiac St., Bldg. 40, Wallingford CT 06492
  
  
 ---------------------------------------------------------------
 Warning -- This program may not be used in any manner that
 competes with the business of Gaussian, Inc. or will provide
 assistance to any competitor of Gaussian, Inc.  The licensee
 of this program is prohibited from giving any competitor of
 Gaussian, Inc. access to this program.  By using this program,
 the user acknowledges that Gaussian, Inc. is engaged in the
 business of creating and licensing software in the field of
 computational chemistry and represents and warrants to the
 licensee that it is not a competitor of Gaussian, Inc. and that
 it will not use this program in any manner prohibited above.
 ---------------------------------------------------------------
  

 Cite this work as:
 Gaussian 16, Revision C.01,
 M. J. Frisch, G. W. Trucks, H. B. Schlegel, G. E. Scuseria, 
 M. A. Robb, J. R. Cheeseman, G. Scalmani, V. Barone, 
 G. A. Petersson, H. Nakatsuji, X. Li, M. Caricato, A. V. Marenich, 
 J. Bloino, B. G. Janesko, R. Gomperts, B. Mennucci, H. P. Hratchian, 
 J. V. Ortiz, A. F. Izmaylov, J. L. Sonnenberg, D. Williams-Young, 
 F. Ding, F. Lipparini, F. Egidi, J. Goings, B. Peng, A. Petrone, 
 T. Henderson, D. Ranasinghe, V. G. Zakrzewski, J. Gao, N. Rega, 
 G. Zheng, W. Liang, M. Hada, M. Ehara, K. Toyota, R. Fukuda, 
 J. Hasegawa, M. Ishida, T. Nakajima, Y. Honda, O. Kitao, H. Nakai, 
 T. Vreven, K. Throssell, J. A. Montgomery, Jr., J. E. Peralta, 
 F. Ogliaro, M. J. Bearpark, J. J. Heyd, E. N. Brothers, K. N. Kudin, 
 V. N. Staroverov, T. A. Keith, R. Kobayashi, J. Normand, 
 K. Raghavachari, A. P. Rendell, J. C. Burant, S. S. Iyengar, 
 J. Tomasi, M. Cossi, J. M. Millam, M. Klene, C. Adamo, R. Cammi, 
 J. W. Ochterski, R. L. Martin, K. Morokuma, O. Farkas, 
 J. B. Foresman, and D. J. Fox, Gaussian, Inc., Wallingford CT, 2019.
 
 ******************************************
 Gaussian 16:  ES64L-G16RevC.01  3-Jul-2019
                11-Nov-2020 
 ******************************************
 --------------------------------
 # b3lyp/6-311++g(d,p) output=wfn
 --------------------------------
 1/38=1,172=1/1;
 2/12=2,17=6,18=5,40=1/2;
 3/5=4,6=6,7=1111,11=2,25=1,30=1,74=-5/1,2,3;
 4//1;
 5/5=2,38=5/2;
 6/7=2,8=2,9=2,10=2,28=1/1;
 99/5=1,6=100,9=1/99;
 -------------------
 Single Point Energy
 -------------------
 Symbolic Z-matrix:
 Charge = -1 Multiplicity = 1
 C                    -0.60254  -0.00006   0.00021 
 H                    -0.82525  -0.89429  -0.55694 
 H                    -0.82528   0.92957  -0.4956 
 H                    -0.82514  -0.03548   1.05324 
 Br                    1.5939    0.       -0.00001 
 Cl                   -3.31903   0.00002  -0.00007 
 
                          Input orientation:                          
 ---------------------------------------------------------------------
 Center     Atomic      Atomic             Coordinates (Angstroms)
 Number     Number       Type             X           Y           Z
 ---------------------------------------------------------------------
      1          6           0       -0.602535   -0.000061    0.000213
      2          1           0       -0.825251   -0.894290   -0.556935
      3          1           0       -0.825280    0.929568   -0.495599
      4          1           0       -0.825141   -0.035479    1.053241
      5         35           0        1.593902    0.000003   -0.000011
      6         17           0       -3.319029    0.000021   -0.000070
 ---------------------------------------------------------------------
                    Distance matrix (angstroms):
                    1          2          3          4          5
     1  C    0.000000
     2  H    1.076876   0.000000
     3  H    1.076873   1.824889   0.000000
     4  H    1.076882   1.824890   1.824889   0.000000
     5  Br   2.196437   2.638603   2.638587   2.638630   0.000000
     6  Cl   2.716494   2.707179   2.707100   2.707435   4.912931
                    6
     6  Cl   0.000000
 This structure is nearly, but not quite of a higher symmetry.
 Consider Symm=Loose if the higher symmetry is desired.
 Stoichiometry    CH3BrCl(1-)
 Framework group  C1[X(CH3BrCl)]
 Deg. of freedom    12
 Full point group                 C1      NOp   1
 Largest Abelian subgroup         C1      NOp   1
 Largest concise Abelian subgroup C1      NOp   1
                         Standard orientation:                         
 ---------------------------------------------------------------------
 Center     Atomic      Atomic             Coordinates (Angstroms)
 Number     Number       Type             X           Y           Z
 ---------------------------------------------------------------------
      1          6           0       -0.481851    0.019927    0.098619
      2          1           0       -0.543757   -0.535151    1.019335
      3          1           0       -0.906161   -0.458937   -0.767582
      4          1           0       -0.649620    1.080703    0.177889
      5         35           0        1.668284   -0.068296   -0.341233
      6         17           0       -3.141136    0.128482    0.642459
 ---------------------------------------------------------------------
 Rotational constants (GHZ):         150.5770648           0.8510889           0.8510889
 Standard basis: 6-311++G(d,p) (5D, 7F)
 There are   126 symmetry adapted cartesian basis functions of A   symmetry.
 There are   121 symmetry adapted basis functions of A   symmetry.
   121 basis functions,   210 primitive gaussians,   126 cartesian basis functions
    31 alpha electrons       31 beta electrons
       nuclear repulsion energy       175.2940838141 Hartrees.
 NAtoms=    6 NActive=    6 NUniq=    6 SFac= 1.00D+00 NAtFMM=   60 NAOKFM=F Big=F
 Integral buffers will be    131072 words long.
 Raffenetti 2 integral format.
 Two-electron integral symmetry is turned on.
 One-electron integrals computed using PRISM.
 NBasis=   121 RedAO= T EigKep=  8.29D-04  NBF=   121
 NBsUse=   121 1.00D-06 EigRej= -1.00D+00 NBFU=   121
 ExpMin= 3.50D-02 ExpMax= 4.40D+05 ExpMxC= 1.51D+04 IAcc=3 IRadAn=         5 AccDes= 0.00D+00
 Harris functional with IExCor=  402 and IRadAn=       5 diagonalized for initial guess.
 HarFok:  IExCor=  402 AccDes= 0.00D+00 IRadAn=         5 IDoV= 1 UseB2=F ITyADJ=14
 ICtDFT=  3500011 ScaDFX=  1.000000  1.000000  1.000000  1.000000
 FoFCou: FMM=F IPFlag=           0 FMFlag=      100000 FMFlg1=           0
         NFxFlg=           0 DoJE=T BraDBF=F KetDBF=T FulRan=T
         wScrn=  0.000000 ICntrl=       500 IOpCl=  0 I1Cent=   200000004 NGrid=           0
         NMat0=    1 NMatS0=      1 NMatT0=    0 NMatD0=    1 NMtDS0=    0 NMtDT0=    0
 Petite list used in FoFCou.
 Keep R1 ints in memory in canonical form, NReq=29147525.
 Requested convergence on RMS density matrix=1.00D-08 within 128 cycles.
 Requested convergence on MAX density matrix=1.00D-06.
 Requested convergence on             energy=1.00D-06.
 No special actions if energy rises.
 SCF Done:  E(RB3LYP) =  -3074.39137318     A.U. after   11 cycles
            NFock= 11  Conv=0.98D-08     -V/T= 2.0014

 **********************************************************************

            Population analysis using the SCF Density.

 **********************************************************************

 Orbital symmetries:
       Occupied  (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A)
       Virtual   (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A)
 The electronic state is 1-A.
 Alpha  occ. eigenvalues -- -482.66302-101.27555 -62.31595 -56.13551 -56.13278
 Alpha  occ. eigenvalues --  -56.13278 -10.08283  -9.18891  -8.52497  -6.94816
 Alpha  occ. eigenvalues --   -6.94665  -6.94665  -6.35366  -6.34477  -6.34477
 Alpha  occ. eigenvalues --   -2.46493  -2.46248  -2.46248  -2.45545  -2.45545
 Alpha  occ. eigenvalues --   -0.62632  -0.54746  -0.51850  -0.30024  -0.30024
 Alpha  occ. eigenvalues --   -0.20788  -0.13904  -0.13904  -0.07409  -0.07358
 Alpha  occ. eigenvalues --   -0.07358
 Alpha virt. eigenvalues --    0.09603   0.11233   0.14628   0.14628   0.15634
 Alpha virt. eigenvalues --    0.17688   0.18163   0.18164   0.20924   0.21961
 Alpha virt. eigenvalues --    0.21961   0.23942   0.23942   0.24807   0.31423
 Alpha virt. eigenvalues --    0.31600   0.31601   0.35248   0.40028   0.51236
 Alpha virt. eigenvalues --    0.53558   0.53559   0.55081   0.55081   0.64428
 Alpha virt. eigenvalues --    0.66566   0.66567   0.70129   0.75264   0.75537
 Alpha virt. eigenvalues --    0.75537   0.79187   0.79187   0.83646   0.83646
 Alpha virt. eigenvalues --    0.85039   0.92966   1.10411   1.13095   1.13095
 Alpha virt. eigenvalues --    1.13533   1.13534   1.15417   1.22743   1.22744
 Alpha virt. eigenvalues --    1.31215   1.54851   1.65385   1.65385   1.74795
 Alpha virt. eigenvalues --    1.89672   1.89672   2.03161   2.13637   2.13637
 Alpha virt. eigenvalues --    2.26572   2.26572   2.29595   2.50988   2.50992
 Alpha virt. eigenvalues --    2.57894   2.63461   2.63594   2.63594   2.73130
 Alpha virt. eigenvalues --    2.73131   3.05636   3.20596   3.85374   3.85375
 Alpha virt. eigenvalues --    4.30035   4.30035   4.36219   4.36219   4.60482
 Alpha virt. eigenvalues --    6.73785   7.71219   7.71221   7.81758   9.87971
 Alpha virt. eigenvalues --   24.04264  26.07197  26.07199  26.25236  48.03612
 Alpha virt. eigenvalues --  215.85103 289.91299 289.91299 290.023391020.81087
          Condensed to atoms (all electrons):
               1          2          3          4          5          6
     1  C    5.048024   0.414726   0.414733   0.414705   0.077393  -0.060595
     2  H    0.414726   0.522854  -0.033517  -0.033518  -0.048971  -0.002465
     3  H    0.414733  -0.033517   0.522847  -0.033518  -0.048973  -0.002467
     4  H    0.414705  -0.033518  -0.033518   0.522879  -0.048962  -0.002460
     5  Br   0.077393  -0.048971  -0.048973  -0.048962  35.492082   0.007318
     6  Cl  -0.060595  -0.002465  -0.002467  -0.002460   0.007318  17.864457
 Mulliken charges:
               1
     1  C   -0.308986
     2  H    0.180890
     3  H    0.180895
     4  H    0.180874
     5  Br  -0.429886
     6  Cl  -0.803788
 Sum of Mulliken charges =  -1.00000
 Mulliken charges with hydrogens summed into heavy atoms:
               1
     1  C    0.233674
     5  Br  -0.429886
     6  Cl  -0.803788
 Electronic spatial extent (au):  <R**2>=           1144.7987
 Charge=             -1.0000 electrons
 Dipole moment (field-independent basis, Debye):
    X=              8.0746    Y=             -0.3299    Z=             -1.6514  Tot=              8.2483
 Quadrupole moment (field-independent basis, Debye-Ang):
   XX=            -90.8859   YY=            -43.8568   ZZ=            -45.7486
   XY=              1.9268   XZ=              9.6350   YZ=             -0.3941
 Traceless Quadrupole moment (field-independent basis, Debye-Ang):
   XX=            -30.7221   YY=             16.3070   ZZ=             14.4152
   XY=              1.9268   XZ=              9.6350   YZ=             -0.3941
 Octapole moment (field-independent basis, Debye-Ang**2):
  XXX=            187.0085  YYY=             -1.5005  ZZZ=            -13.4676  XYY=             20.3029
  XXY=             -6.0479  XXZ=            -30.0772  XZZ=             25.2808  YZZ=             -1.9676
  YYZ=             -4.0282  XYZ=              0.8763
 Hexadecapole moment (field-independent basis, Debye-Ang**3):
 XXXX=          -1798.2229 YYYY=            -72.2705 ZZZZ=           -127.1723 XXXY=             43.2254
 XXXZ=            215.7888 YYYX=             26.9287 YYYZ=             -5.4974 ZZZX=            141.0101
 ZZZY=             -6.1543 XXYY=           -248.8974 XXZZ=           -273.0768 YYZZ=            -33.1860
 XXYZ=             -4.8258 YYXZ=             45.9561 ZZXY=             10.4080
 N-N= 1.752940838141D+02 E-N=-7.702484484793D+03  KE= 3.069974894968D+03
 Unable to Open any file for archive entry.
 1\1\GINC-NODE531\SP\RB3LYP\6-311++G(d,p)\C1H3Br1Cl1(1-)\V69787FF\11-No
 v-2020\0\\# b3lyp/6-311++g(d,p) output=wfn\\Single Point Energy\\-1,1\
 C,0,-0.602535,-0.000061,0.000213\H,0,-0.825251,-0.89429,-0.556935\H,0,
 -0.82528,0.929568,-0.495599\H,0,-0.825141,-0.035479,1.053241\Br,0,1.59
 3902,0.000003,-0.000011\Cl,0,-3.319029,0.000021,-0.00007\\Version=ES64
 L-G16RevC.01\State=1-A\HF=-3074.3913732\RMSD=9.752e-09\Dipole=3.245132
 2,-0.0000651,0.0002253\Quadrupole=-24.3648829,12.1824696,12.1824133,0.
 0002828,-0.0008926,-0.0000066\PG=C01 [X(C1H3Br1Cl1)]\\@
 The archive entry for this job was punched.

 Writing a WFN file to "frame.wfn"


 THE MORE POWERFUL THE METHOD, THE MORE CATASTROPHIC THE ERRORS.
     -- M.D. KAMEN
 Job cpu time:       0 days  0 hours  0 minutes 25.0 seconds.
 Elapsed time:       0 days  0 hours  0 minutes 12.8 seconds.
 File lengths (MBytes):  RWF=     10 Int=      0 D2E=      0 Chk=      2 Scr=      1
 Normal termination of Gaussian 16 at Wed Nov 11 18:34:04 2020.
